crux-test/test/crux/java_api_test.clj

```diff
@@ -130,3 +131,11 @@
         (t/is (thrown-with-msg? IllegalStateException
                                 #"Crux node is closed"
                                 (.db node)))))))
+
+(t/deftest test-history-options
+  (let [asc-keyword (his/->history-options :asc)
+        asc-enum (his/->history-options HistoryOptions$SortOrder/ASC)
+        desc-keyword (his/->history-options :desc)
+        desc-enum (his/->history-options HistoryOptions$SortOrder/DESC)]
+    (t/is (= asc-keyword asc-enum))
+    (t/is (= desc-keyword desc-enum))))
```

Comment:
idiom: we don't tend to explicitly let-bind a name if the definition is small enough and only used once. 

also, possible pedantry, but you're relying on `(his/->history-options :asc)` returning the correct thing :) 

```suggestion
(t/deftest test-history-options
  (t/is (= :asc (his/->history-options :asc)))
  (t/is (= :asc (his/->history-options HistoryOptions$SortOrder/ASC)))
  (t/is (= :desc (his/->history-options :desc)))
  (t/is (= :desc (his/->history-options HistoryOptions$SortOrder/DESC))))
```